doc/esmvalcore/recipe.rst

```diff
@@ -1,30 +1,39 @@
 .. _recipe:
 
-*****************
-ESMValTool recipe
-*****************
+******
+Recipe
+******
 
-Recipes are the instructions telling ESMValTool about the user who wrote the
-recipe, the datasets which need to be run, the preprocessors that need to be
+Overview
+========
+
+After ``config-user.yml``, the ``recipe.yml`` file the second the user inputted file
+to ESMValTool at each run time point. Rceipes contain the data and data analysis
+information and instructions needed to run the diagnostic(s), as well as specific
+diagnostic-related instructions.
+
+Broadly, recipes contain information on the user who wrote the
+recipe file, the datasets which need to be run, the preprocessors that need to be
 applied, and the diagnostics which need to be run over the preprocessed data.
-This information is provided to ESMValTool in the recipe sections:
-`Documentation`_, `Datasets`_, `Preprocessors`_ and `Diagnostics`_,
+This information is provided to ESMValTool in for main recipe sections:
+Documentation_, Datasets_, Preprocessors_ and Diagnostics_,
 respectively.
 
+.. _Documentation:
 
-Documentation
-=============
+Recipe section: ``documentation``
+=================================
 
 The documentation section includes:
 
-- The recipe's author's user name
-- A description of the recipe
-- The user name of the maintainer
-- A list of scientific references
-- the project or projects associated with the recipe.
+- The recipe's author's user name (``authors``, as they appaer in ``config-references.yml`` config-ref_)
+- A description of the recipe (``description``, written in MarkDown format)
+- The user name of the maintainer (``maintainer``, as they appaer in ``config-references.yml`` config-ref_)
```

Comment:
Drop this, we are not going to use it for long and not all recipes contain this information.